CONTRIBUTING.md

```diff
@@ -163,6 +170,20 @@ For votes regarding non-coding issues such as procedure changes, the requirement
 that a veto is accompanied by a *technical* justification is relaxed somewhat,
 though a well reasoned justification must still be included.
 
+### Reversions
+
+A *-1* vote on any recently merged pull request is grounds for an immediate
```

Comment:
Good catch, that was my intention. 

I'll leave open for comments a while yet, then per procedures will open a new PR with that change included.